src/com/tony/server/WorkerThread.java

```diff
@@ -37,6 +39,21 @@ public void run() {
         }
     }
 
+    private void logRequest(String requestMessage) {
+        logger.log("Received Request:\r\n");
+        logger.log(requestMessage);
+    }
+
+    public void logResponse(Response response, byte[] responseMessage) {
+        if(response.getClass().equals(ImageContentResponse.class)){
+            logger.log(ResponseBuilder.buildResponse(
+                    response.getStatus(), response.getHeaders(), "Image body not shown in log"));
+        } else {
+            logger.log("Sending Response:\r\n");
+            logger.log(new String(responseMessage));
+        }
+    }
```

Comment:
if different types of responses have different text representations, this method could get pretty big